test/view.js

```diff
@@ -0,0 +1,397 @@
+      expect(model.get('filteredFruits')).to.eql(expectedFruits);
+    });    
+  }); 
+
+  describe('Updating item in collection', function() {
+     it('adds updated item to view as conditions ARE now satisfied', function() {
+      var model    = (new Model).at('_page');
+      
+      model = addData(model); // Add items to collection
+      model.fn('yellowFruits', function(emit, fruit) {
+        if (fruit.color === 'yellow') {
+          emit(fruit.name + '*' + fruit.color + '*' + fruit.amount*2, '_page.fruits.' + fruit.id);
+        }
+      });
+
+      // Before updating item
+      var view = model.at('fruits').view('yellowFruits');
+      view.ref('_page.filteredFruits');
+      var expectedFruits = createExpectedResult(model, ['bananaId', 'lemonId']);
+      expect(model.get('filteredFruits')).to.eql(expectedFruits);
+
+      model.set('fruits.appleId.color', 'yellow');  // Update item
+
+      // After updating, item is now included in the view
+      view = model.at('fruits').view('yellowFruits');
+      view.ref('_page.filteredFruits');
+      expectedFruits = createExpectedResult(model, ['appleId', 'bananaId', 'lemonId']);
+      expect(model.get('filteredFruits')).to.eql(expectedFruits);
+    });
+
+    it('removes item from view as conditions are NO longer satisfied', function() {
+      var model    = (new Model).at('_page');
+      
+      model = addData(model); // Add items to collection
+      model.fn('yellowFruits', function(emit, fruit) {
+        if (fruit.color === 'yellow') {
+          emit(fruit.name + '*' + fruit.color + '*' + fruit.amount*2, '_page.fruits.' + fruit.id);
+        }
+      });
+
+      // Before updating item included in view
+      var view = model.at('fruits').view('yellowFruits');
+      view.ref('_page.filteredFruits');
+      var expectedFruits = createExpectedResult(model, ['bananaId', 'lemonId']);
+      expect(model.get('filteredFruits')).to.eql(expectedFruits);
+     
+      model.set('fruits.bananaId.color', 'green'); // Update item
+
+      // After updating the item, it is now excluded from the view
+      view = model.at('fruits').view('yellowFruits');
+      view.ref('_page.filteredFruits');
+      var expectedFruits = createExpectedResult(model, ['lemonId']);
+      expect(model.get('filteredFruits')).to.eql(expectedFruits);
+    });
+  });
+});
+
+describe('Derby-View: Events', function() {
+    describe('Adding new item to collection', function() {
+      it('View is empty: triggers "change" event', function() {
+        var model    = (new Model).at('_page');
+
+        model.fn('yellowFruits', function(emit, fruit) {
+          if (fruit.color === 'yellow') {
+            emit(fruit.name + '*' + fruit.color + '*' + fruit.amount*2, '_page.fruits.' + fruit.id);
+          }
+        });
+        var view = model.at('fruits').view('yellowFruits');
+        view.ref('_page.filteredFruits');
+
+        model.on('change', '**', addListener); // Set up event listener
```

Comment:
In my opinion, a clearer/more dense/concise way of writing listeners would be something like this (since you're not re-using the addListener):
model.on('change', '**', function (...) {...});